File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter05/JAVASCRIPT/JS/lettersLowerCase.js
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 05.-Flujo de control II: Estructuras repetitivas
    Problema 05.-imprimir letras minúsculas del alfabeto
    Pagina 168
*/


function calculate(){
    let letter=97;
    var varSpan=document.createElement("span");
    varSpan.setAttribute("id", "idSpan");      

    if(document.getElementById("idBtnCalculate").value=="Ejecutar"){  
        do{
            varSpan.innerHTML+=String.fromCharCode(letter)+" ";
            letter++;
        }while(letter<=122);

        document.getElementById("idLblMessage").appendChild(varSpan);    
        document.getElementById("idBtnCalculate").value="Limpiar";
    }else{
        
        document.getElementById("idSpan").remove();

        document.getElementById("idBtnCalculate").value="Ejecutar";
    }    
}

File: FundamentosDeLaProgramacionLuisJoyanes4taEdicion/chapter05/CPP/lettersLowerCase.cpp
/*Libro:Fundamentos de Programación, Algoritmos, estructura de datos y objetos 4ta edición. 
    Capitulo 05.-Flujo de control II: Estructuras repetitivas
    Problema 05.-imprimir letras minúsculas del alfabeto
    Pagina 168
*/
#include<iostream>
using namespace std;
int main(){
    char letter='a';

    cout<<"Alfabeto en minuscula: \n\n";
    do{
        cout<<letter<<" ";
        letter++;
    }while(letter<='z');
    cout<<"\n";
}
